manifests/b/Blinue/Magpie/0.9.1/Blinue.Magpie.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.6 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json
+
+PackageIdentifier: Blinue.Magpie
+PackageVersion: 0.9.1
+ReleaseDate: 2022-07-25
+InstallerType: zip
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

Reply:
Hi @Trenly,

The package requires this dependency to run:
![image](https://user-images.githubusercontent.com/114952565/217350590-eb76630c-840d-47f3-a58b-ecc8ecace274.png)

However, it gives a different message with the dependency installed:
![image](https://user-images.githubusercontent.com/114952565/217350505-222cfebe-f934-4b53-aacd-d4e90deb351f.png)